doc/explanation/containers_and_vms.md

```diff
@@ -3,22 +3,37 @@
 
 Incus provides support for two different types of {ref}`instances <expl-instances>`: *system containers* and *virtual machines*.
 
-When running a system container, Incus simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
+When running a system container, Incus simulates a virtual version of a full operating system.  
+To do this, it uses the functionality provided by the kernel running on the host system.
 
-When running a virtual machine, Incus uses the hardware of the host system, but the kernel is provided by the virtual machine. Therefore, virtual machines can be used to run, for example, a different operating system.
+When running a virtual machine, Incus uses the hardware of the host system, and the kernel is provided by the virtual machine.  
+Therefore, virtual machines can be used to run, for example, a different operating system.
```

Comment:
The original phrasing with `, but the kernel is provided` sounds better to me.
Same comment as above about adding a line break, doesn't make a difference to the rendered document but makes it harder for us to keep up with changes in LXD.